server/utils/personUtils.test.ts

```diff
@@ -1,73 +1,116 @@
 import PersonUtils from './personUtils'
 import { personFactory, prisonerFactory } from '../testutils/factories'
+import type { Prisoner } from '@prisoner-search'
 
 describe('PersonUtils', () => {
   describe('personFromPrisoner', () => {
+    let prisoner: Prisoner
+
+    beforeEach(() => {
+      prisoner = prisonerFactory.build({
+        bookingId: '1234567890',
+        conditionalReleaseDate: '2024-10-31',
+        dateOfBirth: '1971-07-05',
+        ethnicity: 'White',
+        firstName: 'Del',
+        gender: 'Male',
+        homeDetentionCurfewEligibilityDate: '2025-10-31',
+        indeterminateSentence: false,
+        lastName: 'Hatton',
+        paroleEligibilityDate: '2026-10-31',
+        prisonName: 'HMP Hewell',
+        prisonerNumber: 'ABC1234',
+        religion: 'Christian',
+        sentenceExpiryDate: '2027-10-31',
+        sentenceStartDate: '2010-10-31',
+        tariffDate: '2028-10-31',
+      })
+    })
+
     describe('when all fields are present on a Prisoner Search "Prisoner"', () => {
       it('returns a "Person" with those fields', () => {
-        const prisoner = prisonerFactory.build({
-          bookingId: '1234567890',
-          dateOfBirth: '1971-07-05',
-          ethnicity: 'White',
-          firstName: 'Del',
-          gender: 'Male',
-          lastName: 'Hatton',
-          prisonName: 'HMP Hewell',
-          prisonerNumber: 'ABC1234',
-          religion: 'Christian',
-          sentenceStartDate: '2010-10-31',
-        })
-
         expect(PersonUtils.personFromPrisoner(prisoner)).toEqual({
           bookingId: '1234567890',
+          conditionalReleaseDate: '2024-10-31',
           currentPrison: 'HMP Hewell',
           dateOfBirth: '5 July 1971',
+          earliestReleaseDate: '2026-10-31',
           ethnicity: 'White',
           gender: 'Male',
+          homeDetentionCurfewEligibilityDate: '2025-10-31',
           name: 'Del Hatton',
+          paroleEligibilityDate: '2026-10-31',
           prisonNumber: 'ABC1234',
           religionOrBelief: 'Christian',
+          sentenceExpiryDate: '2027-10-31',
           sentenceStartDate: '2010-10-31',
           setting: 'Custody',
+          tariffDate: '2028-10-31',
+        })
+      })
+    })
+
+    describe('earliestReleaseDate', () => {
+      describe('when the prisoner is on an indeterminate sentence', () => {
+        it('uses `tariffDate`', () => {
+          prisoner = { ...prisoner, indeterminateSentence: true }
+
+          expect(PersonUtils.personFromPrisoner(prisoner).earliestReleaseDate).toEqual(prisoner.tariffDate)
+        })
+      })
+
+      describe('when the prisoner is on a determinate sentence', () => {
```

Comment:
Let's manually set `prisoner = { ...prisoner, indeterminateSentence: false }` in this test, rather than relying on the `prisoner` variable declared at top level to make the state a bit clearer here.